interface-definitions/interfaces-tinc.xml.in

```diff
@@ -0,0 +1,451 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="interfaces">
+    <children>
+      <tagNode name="tinc" owner="${vyos_conf_scripts_dir}/interfaces-tinc.py">
+        <properties>
+          <help>Tinc VPN Tunnel Interface</help>
```

Comment:
See below: let's made all instances of "tinc" lowercase as per the official spelling from the website.

Reply:
done